2do_be/models/task-services.js

```diff
@@ -64,17 +64,9 @@ async function deleteTask(id) {
   }
 }
 
-//Need to fix this up to align with our new approach of updating all fields
-//when updating any field in the task
-
-async function updateTask(id, status, title, description, dueDate) {
+async function updateTask(id, body) {
```

Comment:
I think either way will work! The body contains all of those elements. From what I can tell we can pass as many of the parameters as we want from the front end - so just the status or just the date or all of them. And it will replace only what we pass through.

Reply:
I was confused too but I guess it makes sense now on what req.body does. Thanks for explaining!